examples/learning_graph_embedding_and_predicting.py

```diff
@@ -56,7 +56,7 @@ def main():
     )
 
     kmeans.fit(X=embeddings)
-    centroids = kmeans.centroids_
+    centroids = kmeans.cluster_centers_
```

Comment:
NIT: in that case, maybe `cluster_centers = kmeans.cluster_centers_` would make more sense.